src/msw/listctrl.cpp

```diff
@@ -875,6 +879,25 @@ bool wxListCtrl::GetItem(wxListItem& info) const
     return success;
 }
 
+// Check if the item is visible
+bool wxListCtrl::IsVisible(long item) const
+{
+    bool result = ::SendMessage( GetHwnd(), LVM_ISITEMVISIBLE, (WPARAM) item, 0 );
+    if( result )
+    {
+        HWND hwndHeader = (HWND) ::SendMessage( GetHwnd(), LVM_GETHEADER, 0, 0 );
+        wxRect itemRect;
+        RECT headerRect;
+		GetItemRect( item, itemRect );
```

Comment:
`itemRect` is only used inside the if-statement, so it (and `GetItemRect`) can be moved there.